cmd/types.go

```diff
@@ -11,44 +11,86 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// CronJob represents the same type from the v1beta1 version of the k8s API.
 type CronJob = batchv1beta1.CronJob
+
+// DaemonSet represents the same type from the v1beta1 version of the k8s API.
 type DaemonSet = extensionsv1beta1.DaemonSet
+
+// NetworkPolicy represents the same type from the v1 version of the k8s API.
 type NetworkPolicy = networking.NetworkPolicy
+
+// Pod represents the same type from the v1 version of the k8s API.
 type Pod = apiv1.Pod
+
+// ReplicationController represents the same type from the v1 version of the k8s API.
 type ReplicationController = apiv1.ReplicationController
+
+// SecurityContext represents the same type from the v1 version of the k8s API.
 type SecurityContext = apiv1.SecurityContext
+
+// StatefulSet represents the same type from the v1beta1 version of the k8s API.
 type StatefulSet = v1beta1.StatefulSet
 
+// ObjectMeta represents the same type from the v1 version of the k8s API.
 type ObjectMeta = metav1.ObjectMeta
+
+// PodSpec represents the same type from the v1 version of the k8s API.
 type PodSpec = apiv1.PodSpec
 
+// DaemonSetList represents the same type from the v1beta1 version of the k8s API.
 type DaemonSetList = extensionsv1beta1.DaemonSetList
+
+// DeploymentList represents the same type from the v1beta1 version of the k8s API.
 type DeploymentList = v1beta1.DeploymentList
+
+// NamespaceList represents the same type from the v1 version of the k8s API.
 type NamespaceList = apiv1.NamespaceList
+
+// NetworkPolicyList represents the same type from the v1 version of the k8s API.
 type NetworkPolicyList = networking.NetworkPolicyList
+
+// PodList represents the same type from the v1 version of the k8s API.
 type PodList = apiv1.PodList
+
+// ReplicationControllerList represents the same type from the v1 version of the k8s API.
 type ReplicationControllerList = apiv1.ReplicationControllerList
+
+// StatefulSetList represents the same type from the v1beta1 version of the k8s API.
```

Comment:
Better wording: StatefulSetList is a type alias for v1beta1 version of the k8s API. Same goes for rest of them.